source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardTemplate.cs

```diff
@@ -4,6 +4,7 @@
 using Antlr4.Runtime.Tree;
 using Newtonsoft.Json;
 using System;
+using System.Collections;
```

Comment:
should we create a new branch like release/22.05 based on release/21.09, then cherry-pick?

Reply:
I think we are adding to this branch since it was the last major release, and these changes are only bumping the minor version